app/bundles/CoreBundle/Twig/Helper/AssetsHelper.php

```diff
@@ -696,22 +698,21 @@ public function shortenText($text, $charCount = null)
     }
 
     /**
-     * @param string    $country
-     * @param bool|true $urlOnly
-     * @param string    $class
-     *
-     * @return string
+     * Returns the country flag image or false if not found.
      */
-    public function getCountryFlag($country, $urlOnly = true, $class = '')
+    public function getCountryFlag(string $country, bool $urlOnly = true, string $class = ''): string|bool
```

Comment:
1. Why is the return type different from the `AssetExtension::getCountryFlag()`? Wouldn't it be better to keep the return type for constancy?
2. Why is the new `Default.jpg` image added but it seems it cannot be used according to the unit tests. I'd expect that it will be used instead of the empty string.
3. Please be careful with modifying public and protected method signatures. It's considered a BC break and won't be merged into a minor or patch versions. There is a chance this will still get into M5.0.0-beta but if not, these types will have to be sadly removed.

Reply:
1. I did restore the signature
2. removed
3. as 1